core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -30,46 +34,51 @@ trait AdminApiService extends UserInfoDirectives {
       delete {
         entity(as[Map[String, String]]) { ownerInfo =>
           complete {
-            userServiceConstructor(userInfo).adminDeleteBillingProject(RawlsBillingProjectName(projectId), ownerInfo)
+            userServiceConstructor(userInfo).adminDeleteBillingProject(RawlsBillingProjectName(projectId), ownerInfo).map(_ => StatusCodes.NoContent)
           }
         }
       }
     } ~
     path("admin" / "project" / "registration") {
       post {
         entity(as[RawlsBillingProjectTransfer]) { xfer =>
-          complete { userServiceConstructor(userInfo).adminRegisterBillingProject(xfer) }
+          complete { userServiceConstructor(userInfo).adminRegisterBillingProject(xfer).map(_ => StatusCodes.Created) }
         }
       }
     } ~
     path("admin" / "project" / "registration" / Segment) { (projectName) =>
       delete {
         entity(as[Map[String, String]]) { ownerInfo =>
-          complete { userServiceConstructor(userInfo).adminUnregisterBillingProjectWithOwnerInfo(RawlsBillingProjectName(projectName), ownerInfo) }
+          complete { userServiceConstructor(userInfo).adminUnregisterBillingProjectWithOwnerInfo(RawlsBillingProjectName(projectName), ownerInfo).map(_ => StatusCodes.NoContent) }
         }
       }
     } ~
     path("admin" / "submissions") {
       get {
-        complete { workspaceServiceConstructor(userInfo).adminListAllActiveSubmissions() }
+        complete { workspaceServiceConstructor(userInfo).adminListAllActiveSubmissions().map(StatusCodes.OK -> _) }
```

Comment:
this applies in a lot of places, unfortunately: isn't `.map(StatusCodes.OK -> _)` extraneous within a `complete`?

```
complete { workspaceServiceConstructor(userInfo).adminListAllActiveSubmissions().map(StatusCodes.OK -> _) }
```
could just be:
```
complete { workspaceServiceConstructor(userInfo).adminListAllActiveSubmissions() }
```
unless I'm missing something - you are closer to this than I am!

I'd be fine merging this PR and then cleaning that up as a follow-on

Reply:
You're totally right- I think it's worth fixing now!